enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -71,10 +73,10 @@ Log forwarding will provide a declarative way to specify the outputs for specifi
 
 * `logs.app` - Container logs generated by user applications running on the platform, excluding infrastructure container applications
 * `logs.infra` - Logs generated by both infrastructure components running on the platform and OKD nodes (e.g. journal logs).  "Infra" applications are defined as any pods which run in namespaces: `openshift*`, `kube*`, `default`.
+* `logs.audit` - Logs generated by the nodes' auditd (/var/log/audit/audit.log), audit logs from the kubeapi-server and the openshift-apiserver. This will not be forwarded by default.
```

Comment:
Do we want to have 2 or 3 different ones?  1) system audit logs are parsed and treated very differently than openshift/kube logs.  2) there may be customers who want to collect openshift audit logs but not system audit logs, and vice versa

Reply:
Well, right now the request is to have the audit logs (in general) forwarded to a secure location. If we do get requests from customers to have different pipelines, we could split them up.
* logs.audit for all the audit logs
* logs.linux.audit for the auditd logs
* logs.k8s.audit for the k8s audit logs
* logs.openshift.audit for the openshift audit logs

What do you think? (of separating this in a further iteration)

Reply:
ok - @sichvoge this is something to put on the roadmap for 4.4